types/hedron/index.d.ts

```diff
@@ -1,172 +1,147 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.3
 
-import * as React from 'react';
+import * as React from "react";
 
-export interface ColumnProps {
-    /**
-     * You can specify component to use specific tag
-     * @default 'div'
-     */
-    tagName?: keyof ElementTagNameMap;
-    /**
-     * Draws all child columns with "bounding boxes" for easy
-     * visualization of the grid. This enables debug mode for all the
-     * children of this component
-     * @default false
-     */
-    debug?: boolean;
-    /**
-     * The amount of horizontal columns this row creates.
-     * @default 12
-     */
-    divisions?: number;
-    /**
-     * If true, disable padding.
-     */
-    fluid?: boolean;
-    /**
-     * Width during xs breakpoint.
-     */
-    xs?: number;
-    /**
-     * Width during sm breakpoint.
-     */
-    sm?: number;
-    /**
-     * Width during md breakpoint.
-     */
-    md?: number;
-    /**
-     * Width during lg breakpoint.
-     */
-    lg?: number;
-    /**
-     * Width of left margin during xs breakpoint.
-     */
-    xsShift?: number;
-    /**
-     * Width of left margin during sm breakpoint.
-     */
-    smShift?: number;
-    /**
-     * Width of left margin during md breakpoint.
-     */
-    mdShift?: number;
-    /**
-     * Width of left margin during lg breakpoint.
-     */
-    lgShift?: number;
+export interface Styleable {
+  /**
+   * The amount of padding added
+   */
+  padding?: string;
+  /**
+   * The amount of padding added
+   */
+  margin?: string;
+  /**
+   * Width Property
+   */
+  width?: string;
+  /**
+   * Height Property
+   */
+  height?: string;
+  /**
+   * Visibilty Property
+   */
+  visibility?: "visible" | "hidden" | "collapse" | "initial" | "inherit";
+  /**
+   * Display Property
+   */
+  display?: string;
+  /**
+   * Opacity Property
+   */
+  opacity?: number;
+  /**
+   * Background Property
+   */
+  background?: string;
+  /**
+   * Border property
+   */
+  border?: string;
+  /**
+   * Controls hidden
+   */
+  hidden?: boolean;
 }
 
-export interface PageProps {
-    /**
-     * You can specify component to use specific tag
-     * @default 'div'
-     */
-    tagName?: keyof ElementTagNameMap;
-    /**
-     * Draws all child columns with "bounding boxes" for easy
-     * visualization of the grid. This enables debug mode for all the
-     * children of this component
-     * @default false
-     */
-    debug?: boolean;
-    /**
-     * Enabling fluid mode disables the fixed width of the Page
-     */
-    fluid?: boolean;
-    /**
-     * If not using fluid, set a custom width for the page.
-     */
-    width?: string;
+export interface ProviderProps {
+  /**
+   * The amount of padding added
+   */
+  padding?: string;
+  /**
+   * Object specifying the breakpoints
+   * { mobile: "-500", tablet: "501-750", wide: "+750" }
```

Comment:
I think adding the [@example](http://usejsdoc.org/tags-example.html) tag should improve the IDE integration in some cases

Reply:
Good point. Will add it